src/components/dashboard/news-item.vue

```diff
@@ -0,0 +1,51 @@
+<script>
+import TimeFormat from '~/mixins/time-format'
+export default {
+  name: 'news-item',
+  mixins: [TimeFormat],
+  components: {
+    Chips: () => import('~/components/common/chips.vue')
+  },
+  props: {
+    // news: () => {} // of { title, date, description, author }
+    news: Object
+  },
+  computed: {
+    tagType () {
+      let color
+      switch (this.news.type) {
+        case 'announcement':
+          color = 'primary'
+          break
+        case 'new feature':
+          color = 'grey-6'
+          break
+        default:
+          color = 'primary'
+          break
+      }
```

Comment:
Instead of this switch statement, I would allow the color be provided via a prop `color: String` instead. This would allow us to easily add new types.